### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by eeswr**

### CODE REVIEW

**Comment by mihaidma** [comment]
could you please emphasize the line that it depends on basic and entity plugins?
other than that LGTM

File: test/example.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+var seneca = require('seneca')()
```
Comment (mcdonnelldean) [inline test/example.js]: @mihaidma @eeswr This should be in docs/examples/basic.js

File: test/example.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+seneca.use(require('seneca-basic'))
+seneca.use(require('seneca-entity'))
+seneca.use(require('seneca-user'))
+
+seneca.ready(function () {
```
Comment (mcdonnelldean) [inline test/example.js]: Change `function ()` to `()`

Comment (mcdonnelldean) [inline test/example.js]: Change to:

```
seneca.act('role:user,cmd:register', {name: "Flann O'Brien", email: 'nincompoop@deselby.com', password: 'blackair'}, (msg, done) =>
```

**Comment by mihaidma** [comment]
Parameter names need rename as Dean pointed out.
A few more arrows can be added in code.
"var" could be replaced with "let"

File: doc/examples/basic.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+var seneca = require('seneca')()
+seneca.use(require('seneca-basic'))
+seneca.use(require('seneca-entity'))
+seneca.use(require('seneca-user'))
+
+seneca.ready((msg, done) => {
+  seneca.act({role: 'user', cmd: 'register', name: "Flann O'Brien", email: 'nincompoop@deselby.com', password: 'blackair'},
+   (msg, done) => {
+     if (msg) {
```
Comment (mcdonnelldean) [inline doc/examples/basic.js]: My bad, it's `err, reply` not `msg, done for seneca.acts`

File: doc/examples/basic.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+var seneca = require('seneca')()
+seneca.use(require('seneca-basic'))
+seneca.use(require('seneca-entity'))
+seneca.use(require('seneca-user'))
+
+seneca.ready((msg, done) => {
```
Comment (mihaidma) [inline doc/examples/basic.js]: ready takes no params

File: doc/examples/basic.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  seneca.act({role: 'user', cmd: 'register', name: "Flann O'Brien", email: 'nincompoop@deselby.com', password: 'blackair'},
```
Comment (mcdonnelldean) [inline doc/examples/basic.js]: @mihaidma Can you run through a sample of using jsonic strings with payloads with @eeswr ? We should have user/cmd as a jsonic string, the rest as payload

File: doc/examples/basic.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
+var seneca = require('seneca')()
+seneca.use(require('seneca-basic'))
```
Comment (mcdonnelldean) [inline doc/examples/basic.js]: These should be 

```
var Seneca = require('seneca')
var Basic = require('seneca-basic')
...

var seneca = Seneca()
  .use(Basic)
  .use(...) // and so forth

```

**Comment by rjrodger** [comment]
Detailed examples now provided by seneca-msg-test unit tests

**Pull request closed without merging**